backend/core/src/Core.Infrastructure/Nexus/Repositories/NexusTransactionRepository.cs

```diff
@@ -109,6 +109,32 @@ public async Task<Paged<Transaction>> GetAsync(string publicKey, int page, int p
             };
         }
 
+        public async Task<Paged<Transaction>> GetByCodeAsync(string code, CancellationToken cancellationToken = default)
+        {
+            int page = 1; // default value
+            int pageSize = 10; // default value
```

Comment:
wouldn't it be more convenient to have this be the max to reduce the nr of requests needed?

Reply:
This is just used as default to return from the repo. It is currently not used in the query, so it won't affect the no of requests needed.